tasmota/xdrv_21_wemo_multi.ino

```diff
@@ -235,8 +235,12 @@ private:
 
   String WemoSerialnumber(void) {
     char serial[18];
-
-    snprintf_P(serial, sizeof(serial), PSTR("201612K%08X-%d"), ESP_getChipId(), _deviceId);
+    if (_deviceId == 1)
+    {
+      snprintf_P(serial, sizeof(serial), PSTR("201612K%08X"), ESP_getChipId());
+    } else {
+      snprintf_P(serial, sizeof(serial), PSTR("201612K%08X-%d"), ESP_getChipId(), _deviceId);
+    }
```

Comment:
What hapens if you change this to 
```
snprintf_P(serial, sizeof(serial), PSTR("201612K%08X%02X"), ESP_getChipId(), _deviceId);
```
so removing the dash and make it hex.

Reply:
I have tried it. This change works fine for me. The if/else is not necessary in both cases.

Reply:
And you can add the device 1 to routines in Alexa app ok?

Still expect there to be issue with dupe devices after upgrade from pre-9.2.0 as the serial/uuid changed.

Reply:
Yes, everithing works fine, now. I had problems adding the devices to a group or routine as not all of them where added to that group. I made several test and when adding a device to that group another device (I expect the one first discovered) was added. Now I can add every new device to every group as I want to. Before I can't add the devices "Terrasse", "Haustuer" and "Garten" to the same group. No it works.

![Screenshot_20201220-131746_Amazon Alexa](https://user-images.githubusercontent.com/35624593/102713187-89ee8480-42c6-11eb-84ff-abc6ec3f444e.jpg)

Reply:
Thx so the latest fix solved both issues.

Closing this PR.

Reply:
I tested with these two changes to remove the if/else, and yes does fix the routine/group issue, but I do still end up with duplicate device upgrading when the device existed in Alexa before.  Both devices still work ok however.